css/index.css

```diff
@@ -1,3 +1,283 @@
+    margin-bottom: 15px;
+}
+
+.container {
+    width: 880px;
+    margin: auto;
+}
+
+
+header {
+    display: inline;
+    width: 75%;
+}
+
+header nav {
+     display: inline;
+    width: 75%;
+}
+
+header nav a {
+    padding: 0 25px 0px;
+    font-size: 12px;
+    color: gray;
+    text-decoration: none;
+}
+
+header nav a:hover {
+    text-decoration: underline;
+    color: black;
+}
+
+header .logo {
+    margin-top: 25px;
+    max-width: 150px
+}
+
+.cta .cta-text {
+  display: inline-block;
+  vertical-align: top;
+  font-size: 55px;
+  margin-top: 100px;
+  text-align: center;
+  padding: 0px 75px;
+}
+
+.cta .cta-text button {
+  background: white;
+  vertical-align: top;
+  margin-top: 6px;
+  padding: 5px 35px;
+  font-size: 13px;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  display: inline-block;
+  margin-top: 69px;
+  max-width: 300px
+}
+
+
+.main-content {
+   border-top: 1px solid black; 
+   border-bottom: 1px solid black;
+   margin: 75px 0 35px 0;
+   padding: 35px 0;
+   font-size: 12px;
+   line-height: 18px;
+}
+
+.main-content .top-content {
+    display: flex;
+}
+
+.main-content .middle-img {
+    display: block;
+    margin: 30px auto;
+}
+
+.main-content .bottom-content {
+    display: flex;
+}
+
+.main-content .bottom-content p {
+    padding-right: 20px;
+}
+
+/* Contact Styles */
+
+.contact  {
+    margin: 15px 0;
+    font-size: 12px;
+}
+
+
+footer {
+    text-align: center;
+    margin: 40px 0 20px;
+    font-size: 12px;
+}
+
+
+@media (max-width: 800px) {
+
+    .container {
+        width: 100%;
+    }
+
+    header {
+        flex-wrap: wrap;
+    }
+  
+    header nav {
+        width: 768px;
+        padding: 0 30px;
+    }
+
+    header .logo {
+        order: -1;
+        display: block;
+        margin: 30px auto 0;
+    }
+
+    .cta {
+        justify-content: center;
+    }
+    
+    .cta .cta-text {
+        margin: 0;
+    }
+
+    .cta img {
+        display: none;
+    }
+
+    .main-content .middle-img {
+       width: 100%;
+    }
+
+    .main-content {
+       margin: 75px 20px 35px;
+    }
+
+    .text-content {
+        margin: 0 10px;
+    }
+
+    .contact {
+        margin: 0 30px;
+    }
+}
+
+@media (max-width: 500px) {
+
+    header {
+        justify-content: center;
+    }
+
+    header nav {
+        width: 100%;
+        flex-direction: column;
+        margin: 0;
+        padding: 0;
+    }
+
+    header nav a {
+        width: 100%;
+        display: block;
+        text-align: center;
+        padding: 20px 0;
+        font-size: 20px;
+        border-bottom: 1px solid gray;
+    }
+
+    header nav a:first-child {
+        border-top: 1px solid gray;
+    }
+
+    header nav a:hover {
+        background: gray;
+    }
+
+    header .logo {
+        margin: 20px 0;
+    }
+
+    .cta {
+        margin-top: 50px;
+        flex-flow: column;
+        align-items: center;
+    }
```

Comment:
You'd want to make sure you set the display to flex here so the flex properties are applied to its children elements.